web/router.ex

```diff
@@ -89,4 +87,11 @@ defmodule CodeCorps.Router do
     resources "/user-roles", UserRoleController, only: [:create, :delete]
     resources "/user-skills", UserSkillController, only: [:create, :delete]
   end
+
+  scope "/", CodeCorps, host: "api." do
+    pipe_through [:api, :bearer_auth, :current_user]
+    get "/:slug", SluggedRouteController, :show
+    get "/:slug/projects", ProjectController, :index
+    get "/:slug/:project_slug", ProjectController, :show
+  end
```

Comment:
Need to be authenticated to `:show` or `:index` on donation goals, so these slug routes match the url before `/donation_goals`  does. Had to move it below. 

If we actually do not have to be authenticated for `:show` and `:index`, then we can move this back up.